platform/ui/src/components/ohifModal/OHIFModal.js

```diff
@@ -29,7 +29,7 @@ const OHIFModal = ({
   const renderHeader = () => {
     return (
       title && (
-        <div className="OHIFModal__header">
+        <div className="OHIFModal__header" data-cy="OHIFModal-header">
```

Comment:
Personal preference, can we keep `data-cy` values lowercase, words separated by dashes?
Also, something more generic like `modal-header` would work well here :+1: